presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -510,6 +594,18 @@ private static WebApplicationException badRequest(String message)
                 .build());
     }
 
+    /**
+     * Returns the given string if is non-null and non-empty.
```

Comment:
I think we need just the summary: Check and return a given string if it is non-null and non-empty.